test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Sql/SchemaTests.cs

```diff
@@ -90,6 +88,41 @@ public async Task WhenRequestingScript_GivenNonIntegerVersion_TheServerShouldRet
             await SendAndVerifyStatusCode(HttpMethod.Get, "_schema/versions/abc/script", HttpStatusCode.NotFound);
         }
 
+        [Fact]
+        public async Task WhenRequestingScript_GivenPostMethod_TheServerShouldReturnNotFound()
+        {
+            await SendAndVerifyStatusCode(HttpMethod.Post, "_schema/versions/1/script", HttpStatusCode.NotFound);
+        }
+
+        [Fact]
+        public async Task WhenRequestingScript_GivenPutMethod_TheServerShouldReturnNotFound()
+        {
+            await SendAndVerifyStatusCode(HttpMethod.Put, "_schema/versions/1/script", HttpStatusCode.NotFound);
+        }
+
+        [Fact]
+        public async Task WhenRequestingScript_GivenDeleteMethod_TheServerShouldReturnNotFound()
+        {
+            await SendAndVerifyStatusCode(HttpMethod.Delete, "_schema/versions/1/script", HttpStatusCode.NotFound);
+        }
+
+        [Fact]
+        public async Task WhenRequestingScript_GivenSchemaIdFound_TheServerShouldReturnScript()
```

Comment:
We should test failure cases too.

Reply:
Failure cases are like - schemaId not found and Verbs except GET.
Yes I  missed to add testcase for schemaId not found- work in progress. While testcases are already added for the failure verbs(POST, PUT, DELETE). 
Please let me know if I am missing anything here. Thanks